msu/hooks/entity/world/player_party.nut

```diff
@@ -70,4 +69,11 @@
 		}
 		return mult;
 	}
+
```

Comment:
This is a comment regarding other stuff in this file that was present before this PR. All the cases where we are checking for `if ("xyz" in abc)` should be resolved in one of the following ways:
1. Use `if (::MSU.isIn("xyz", abc, true))` to account for inheritance and WeakTableRefs.
2. OR add the keys to the base classes with a default return of `1.0` so that we don't have to check for `in` every time. This might be more performant too.

Reply:
Did #2